central/reports/common/report_formatter.go

```diff
@@ -36,7 +36,8 @@ var (
 	}
 )
 
-type imageVulnerability struct {
+// ImageVulnerability data used for generating vuln reports
+type ImageVulnerability struct {
```

Comment:
Why were the changes in this file required?

Reply:
Had to export `Deployment` struct in this file, so it can be used by `groupByClusterAndNamespace` func, but I didn't need to export everything else. I will change to just export Deployment.